Libraries/GhostNetwork.EventBus.RabbitMq/ConnectionProvider.cs

```diff
@@ -20,8 +22,14 @@ public IConnection GetConnection()
         {
             lock (connectionLock)
             {
-                // TODO: Handle BrokerUnreachableException
-                connection ??= connectionFactory.CreateConnection();
+                Policy
+                .Handle<BrokerUnreachableException>()
+                .WaitAndRetryForever(retryAttempt =>
```

Comment:
`WaitAndRetryForever` is OK. But `Math.Pow(2, retryAttempt)` might be insecure
I think it will be better to use delay pattern with upper limit for 1 minute